ledger-tool/src/bigtable.rs

```diff
@@ -53,6 +53,16 @@ async fn upload(
     };
     let blockstore = Arc::new(blockstore);
 
+    let starting_slot = if let Some(starting_slot) = starting_slot {
```

Comment:
This line looks a little silly to me but blanking on a better way to do this. The if/else is necessary as `get_first_available_block()` returns a Result, and you can't `?` a Result in a closure (ie no dice on `starting_slot.unwrap_or_else(|| blockstore.get_first_available_block()?)`;